privacyidea/static/components/config/controllers/periodicTaskController.js

```diff
@@ -51,6 +51,18 @@ myApp.controller("periodicTaskController", ["$scope", "$stateParams", "$state",
             $scope.getPeriodicTasks();
         });
     };
+
+    $scope.confirmDeletePeriodicTask = function (ptaskid) {
```

Comment:
This would be superflous

Reply:
fixed in 9c208ab4212d0e7fd5a0133faf6722115b3a40c9.